cfme/configure/tasks.py

```diff
@@ -160,3 +160,28 @@ def all_other_tasks(**filter_kwargs):
     Returns: List of dicts.
     """
     return _get_tasks("tasks_all_other", **filter_kwargs)
+
+
+def is_vm_analysis_finished(vm_name, clear_tasks_after_success=False):
```

Comment:
Should this live here or in the VM/Instance class? Though it is indeed part of the Tasks page. I think that perhaps it is a method performed on a vm and as such perhaps it should live there instead?
@mfalesni ?